stdlib/area/position.lua

```diff
@@ -201,4 +201,21 @@ function Position.tostring(pos)
     end
 end
 
+--- Returns the opposite direction - Adapted from Factorio util.lua
+-- @param direction the defines.direction to get the opposite of
+-- @return the opposite direction
+function Position.opposite_direction(direction)
+    local opposites = {
```

Comment:
While the performance loss from such a small table creation is negligible. I will make the change to move it outside of the Call.

I actually had a tough time deciding to put this in the Position module.  :) But I suppose an opposite direction is a position.